### PR TITLE
Add pydocstyle check

### PR DESCRIPTION
**Pull request by alexbeatnik**
Due to the fact that the use of "flake8-docstrinds" cause verification errors relate to "pydocstyle",
it makeing the "pydocstyle" check separately from the "flake8" check.
So "pydocstyle" check add to "code-assesment.sh"
Also specify the versions "pydocstyle" and "flake8" add to "requirments-dev.txt" and
"flake8-docstrinds" delete from "requirments-dev.txt"
Since "open" and "load" methods are system methods this methods of the "Page" class renaming
"open" to "loaded" and "load" to "availble".

### CODE REVIEW

**Comment by alexbeatnik** [comment]
@extsoft We still have an error ""module 'pydocstyle' has no attribute 'tokenize_open'" 
https://gitlab.com/pycqa/flake8-docstrings/issues/36 here i finded how to fix it. I use it in this PR

**Comment by alexbeatnik** [comment]
> @alexbeatnik could you please configure separate executions for `pydocstyle` and `flake8`? And please use the latest version of both. We don't need `flake8-docstrinds` anymore as it produces some level of chaos.

Done

File: requirements-dev.txt [hunk 1]
```diff
@@ -2,5 +2,6 @@
 # And these packages have not to be installed to a production environment.
 black==18.9b0
 pylint==2.3.1
-flake8-docstrings==1.3.0
+flake8==3.7.8
 mypy==0.670
+pydocstyle==4.0.0
```
Comment (extsoft) [inline requirements-dev.txt]: @alexbeatnik it's not enough to run `pydocstyle` separately. You have to invoke it explicitly during coding assessment procedure and move related configuration out of flake's config file to the new one.

Comment (alexbeatnik) [reply requirements-dev.txt]: @extsoft If i understend you correct, then done

**Pull request merged**